### ISSUE TITLE
Document status bar android quirks

### ISSUE THREAD

**Issue by brentvatne** [react-navigation/react-navigation.github.io#358]
https://github.com/react-navigation/react-navigation/issues/5568

similarly, if you were to set it to draw-under outside of expo you'd need to adjust it

**Comment by cpboyd** [comment]
@brentvatne using `SafeAreaView.setStatusBarHeight(0)` in Expo with typescript and the latest packages throws a `_reactNative.SafeAreaView.setStatusBarHeight is not a function` error.

Perhaps due to the `hoistNonReactStatic` not copying the static function properly from `SafeView` to `SafeAreaView`?

**Comment by brentvatne** [comment]
it looks like you're importing it from react-native and not react-navigation

**Comment by cpboyd** [comment]
@brentvatne Good catch.  I thought I'd copied your import statements, but it seems I may have been foiled by VS Code's auto-import. 🤦🏼‍♂️
